MapboxNavigation/DayStyle.swift

```diff
@@ -75,6 +75,7 @@ open class DayStyle: Style {
         DistanceLabel.appearance(whenContainedInInstancesOf: [StepInstructionsView.self]).valueTextColor = #colorLiteral(red: 0.431372549, green: 0.431372549, blue: 0.431372549, alpha: 1)
         DistanceRemainingLabel.appearance().font = UIFont.systemFont(ofSize: 18, weight: .medium).adjustedFont
         DistanceRemainingLabel.appearance().normalTextColor = #colorLiteral(red: 0.431372549, green: 0.431372549, blue: 0.431372549, alpha: 1)
+        DraggableView.appearance().gradientColors = [.gray, .lightGray, .gray]
```

Comment:
Nit: use color literals (as above) to make the color scheme easier to visualize and tweak in Xcode.